### PR TITLE
Chinese translation of the Maps Workshop report

### PR DESCRIPTION
**Pull request by xueyuanjia**
Hi @tguild Beihang team developed a Chinese translation of the Maps workshop. Would you like to link it to the English report?

### CODE REVIEW

**Comment by prushforth** [comment]
LGTM! Thanks for doing this translation. I think there should be a link or a blog post linking to the translation.

**Comment by tguild** [comment]
Thanks @xueyuanjia

File: report-zh.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+<h4 id="web-maps-for-cognitive-accessibility">Web地图的认知障碍可访问性</h4>
+
+包括David Fazio、John Kirkwood和John Rochford在内的[专家小组](https://youtu.be/Jb-vLX7a1m0?list=PLNhYw8KaLq2VuEWzz096iRjtvWel-p2AJ&t=2534)讨论了认知障碍群体的地图相关需求。他们都是W3C认知障碍可访问性（COGA）任务小组的成员。COGA任务组的主席Lisa Seeman也参与了专家讨论组的规划工作，但由于日程安排上的沟通失误，她最终未能参加。总之，专家组成员指出，虽然在这一领域还需要进行大量的用户研究，但从广义上讲，针对COGA用户的可访问Web地图技术是一个巨大的市场，这个市场最终也适用于所有人。具体来说，随着大脑（更准确地说是顶叶）的老化，与导向（也就是室内外导航）相关的功能减弱。导向的关键挑战是找到一种方法，利用符合用户认知能力的线索来描述空间信息和方向。这些线索可以是可视化的，也可以是文字的，针对不同的人有不同的形式。该小组还确认，这样的技术不需要局限于专有软件，事实上，他们的特别任务组正在为[COGA用户制定标准](https://w3c.github.io/coga/content-usable/)。希望相关的标准工作组之间能够进行合作，一同为地图用户提出这些需求。
+
+<h4 id="creating-accessible-web-map-widgets">建立无障碍Web地图部件</h4>
+
+本次研讨会汇集了无障碍Web地图这一新兴领域的全球专家和研究人员。他们讨论了残障人士，特别是有视觉障碍的人，使用Web地图会面临的问题，以及Web地图社区可以共同采取哪些行动来促进对地图内容和功能的可访问性。专家组成员包括来自Fizz Studio的Doug Schepers、来自Smith Ketttlewell眼科研究所的工程师Brandon Biggs、来自玛丽皇后大学的Tony Stockman博士和来自缅因大学的Nicholas Giudice博士。
```
Comment (Malvoz) [inline report-zh.md]: s/Smith Ketttlewell/Smith-Kettlewell

File: report-zh.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+Claudia Loitsch和Julian Striegl描述了如何使用地图来改善[室内位置的可访问性](https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Claudia_Loitsch_Julian_Striegl.pdf) 。他们的研究表明，目前室内地图的覆盖率非常低。更重要的是，对于如何让地图和地理数据能够满足不同情景和不同用户需求，目前还没有统一的标准。他们总结，由于室内地图数据的市场预计在未来几年将显著增长，上述问题为制定此类标准提供了绝佳机会，以免在不兼容的解决方案上消耗甚至浪费绘图工作。
+
+物理无障碍是指为身体残疾者提供的设施，如轮椅坡道等。WheelMap.org网站的Sebastian Felix Zappe认为[HTML中的地图可以被用于“残障主流化”](https://www.w3.org/2020/maps/supporting-material-uploads/presentations/Sebastian_Felix_Zappe_Holger_Dieteric.pdf)，也就是说，确保无障碍信息通过标准的制定，成为所有Web地图服务的一部分。
+
+来自Smith-Ketttlewell眼科研究所的Brandon Biggs[强调了向盲人和视力受损用户展示地图内容的重要性](https://www.w3.org/2020/maps/supporting-material-uploads/position-statements/Brandon_Biggs-Smith-Kettlewell.pdf)。WCAG标准和技术目前还没有涵盖Web地图中图形信息的复杂性，也没有涵盖将这种空间信息传达给非可视Web用户的方法。随着浏览器新增的内置功能，比如Web振动、音频和语音识别API，可以为Web创建更高级的非可视化地图。为了生成数字或振感有声地图，地理空间数据集中的每个要素都需要名称和完整的几何特征。Biggs开发的非视觉地图使用了空间碰撞检测，用户操作他们在空间中的虚拟位置，当虚拟位置与要素发生碰撞时，就会读出要素的名称属性。现有的栅格地图和未标记的特征向量集不能提供所需的文本描述。即使是将要素标记为点标记的地图也一样，因为要素在空间中的比例是未知的。Biggs的结论是，如果不考虑非可视地图对Web地图要素信息的数据建模要求，构建标准HTML组件是毫无意义的。这指出了建立Web地图标准需要解决的至少两个关键问题。
```
Comment (Malvoz) [inline report-zh.md]: s/Smith-Ketttlewell/Smith-Kettlewell

**Review by Malvoz — CHANGES_REQUESTED** [review]

File: report-zh.md (hunk 2 above)
Comment (xueyuanjia) [reply report-zh.md]: Sorry for the typos and now fixed. Thanks @Malvoz

**Comment by xueyuanjia** [comment]
> LGTM! Thanks for doing this translation. I think there should be a link or a blog post linking to the translation.

I've also included the link to the original English report in the translation.

**Comment by Malvoz** [comment]
Thanks @xueyuanjia! @prushforth I think we should merge this now.

**Comment by prushforth** [comment]
@Malvoz @xueyuanjia I agree but there will be no link to it. I don't know who should decide on the W3C side how to make alternative translations available now that @tguild has moved on. I am happy to include a link in zh text at the top if you want to do it that way.

**Pull request merged**